docs/userGuide/siteConfiguration.md

```diff
@@ -118,4 +118,8 @@ The ignore pattern follows the [glob pattern used in .gitignore](https://git-scm
 
 **The level of headings to be indexed for searching.** Default: `3` %%i.e., only headings of levels 1,2,3 will be indexed for searching%%.
 
+#### **`disableSearch`**

```

Comment:
I think using a positive term `enableSearch` would be better (this is similar to `searchable` for pages).